Dockerfile

```diff
@@ -43,6 +43,8 @@ CMD [ \
 # Base builder
 FROM base as builder
 COPY docker/keyrings/nodesource.gpg /usr/share/keyrings/nodesource.gpg
+# include .git directory to allow reading version tag from git
+COPY .git/ /app/
```

Comment:
The Vue version used it at build time when building the /dist.  Correct me if I'm wrong but the settings.py is executed at run-time meaning that you need to have a .git directory in the resulting end image folder or parent folder when running the application?

Have you tested the built image to see if the resulting end image has this directory and can read the tag information?

In addition to this it may be nice to add the .git info to the 'dev' tag so people running it can also quickly know what version they're on.

Reply:
I fixed these issues and tested the built image, and everything works as expected for me